### PR TITLE
chore: update custom types directory for new sm format

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
change the custom-types directory for new version of slicemachine, this will require version 4 of the cli (currently in aplha)

### CODE REVIEW

File: custom_types/page/index.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+  "id": "page",
+  "name": "Page",
+  "repeatable": true,
```
Comment (hypervillain) [inline custom_types/page/index.json:4]: It should have an empty "description" and a boolean "status" I believe! See https://documenter.getpostman.com/view/5743666/TzCHBqbH#9d1e3269-ee00-4b7e-94df-a8c092b88f99

File: custom_types/page/index.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  "json": {
+    "Page": {
+      "uid": {
+        "type": "UID",
+        "config": {
+          "label": "UID",
+          "placeholder": "unique-identifier-eg-homepage"
+        }
+      },
+      "body": {
```
Comment (hypervillain) [inline custom_types/page/index.json:14]: Can you rename "body" to "slices"?

**Review by hypervillain — APPROVED** [review]

**Comment by hypervillain** [comment]
Also, `custom_types` should be moved to `customtypes`. This also applies here: https://github.com/prismicio-community/nextjs-sm-tutorial-series/pull/3/files and in the Nuxt tutorial I guesss

**Review by hypervillain — CHANGES_REQUESTED** [review]
We just need to rename the folder and the 2 fields to both cts and it's good to go for me!

File: custom_types/page/index.json (hunk 1 above)
Comment (MarcMcIntosh) [reply custom_types/page/index.json:4]: I'll check 'common.models.masks.Mask#readerForm' and which is use when for creating a repo.

Are these changes going to be consistent across all of prismic?

**Pull request closed without merging**